LabeloTests/BUILD.bazel

```diff
@@ -0,0 +1,21 @@
+load("@build_bazel_rules_apple//apple:ios.bzl", "ios_unit_test")
+load("@build_bazel_rules_swift//swift:swift.bzl", "swift_library")
+
+ios_unit_test(
+    name = "LabeloTests",
+    bundle_id = "app.labelo.LabeloTests",
+    minimum_os_version = "18.0",
+    test_host = "//Labelo:Labelo",
```

Comment:
Try to avoid test host if possible. That way your tests won't need sinulator at all and will be way faster.